src/main/java/Main.java

```diff
@@ -0,0 +1,66 @@
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import model.Pet;
+import model.Shelter;
+import service.PetService;
+import service.PetServiceMain;
+import service.serializer.Format;
+import service.serializer.PetSerializer;
+import view.Command;
+import view.Menu;
+
+import java.util.Optional;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        PetSerializer petSerializer = new PetSerializer(new JsonMapper(), Format.JSON);
+        Shelter shelter = new Shelter();
+
+        try {
+            Optional<Shelter> shelterData = petSerializer.deserialize();
+            if (shelterData.isPresent()) {
+                shelter = shelterData.get();
+            }
```

Comment:
what is the point of using optional here, if you can do the same with null != shelterData